src/extensions/ExtensionManager.as

```diff
@@ -49,7 +49,9 @@ public class ExtensionManager {
 	protected var extensionDict:Object = new Object(); // extension name -> extension record
 	private var justStartedWait:Boolean;
 	private var pollInProgress:Dictionary = new Dictionary(true);
-	static public const wedoExt:String = 'LEGO WeDo';
+	static public const picoBoardExt:String = 'PicoBoard';
+	static public const wedoExt:String = 'LEGO WeDo 1.0';
+	static public const wedo2Ext:String = 'LEGO WeDo 2.0';
```

Comment:
@cwillisf
Won't renaming original WeDo cause opening issues when loading saved files that already have 

"extensionName": "LEGO WeDo"

in their JSON?